core/src/test/resources/sql/misc.iq

```diff
@@ -18,7 +18,75 @@
 !use post
 !set outputformat mysql
 
-# Compared by casting varchar to number
+SELECT UUID '123e4567-e89b-12d3-a456-426655440000';
```

Comment:
What are the results of `cast(null as UUID)` and `UUID null`

Reply:
I will add tests for these. But they should be NULL.

Reply:
the latter is in fact illegal, and won't parse; `UUID` must be followed by a string literal. This is similar to `DATE`, `TIMESTAMP`, and other literals.